src/clj/game/core-io.clj

```diff
@@ -81,10 +81,15 @@
 
 
 ; In-game chat commands
+(defn set-adv-counter [state side target value]
+  (set-prop state side target :advance-counter value)
+  (trigger-event state side :advancement-placed target)
+  (system-msg state side (str "sets advancement counters to " value " on "
```

Comment:
I would prefer that the `system-msg` comes before `trigger-event`, otherwise your messages might end up out of order (for example if we add a message to morph ice).